themes/autumn.toml

```diff
@@ -0,0 +1,33 @@
+# Added by: ArmanDris
```

Comment:
```suggestion
```

This provides no value. Anyone interested in figuring this out can just git blame the file.